packages/shared/src/test_utils/test_common.ts

```diff
@@ -6,7 +6,7 @@ const jsdom = new JSDOM();
 const setURL = (url: string) => {
     jsdom.reconfigure({ url });
     // eslint-disable-next-line no-unused-expressions
-    jsdom.window.location.href === url;
+    jsdom.window.location.href = url;
```

Comment:
Fix for:
<img width="670" alt="Screenshot 2023-07-19 at 3 27 16 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/bea02ada-4553-4a43-860c-548dbcabbcc0">